.github/workflows/main.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           - "3.1"
           - "3.2"
           - ruby-head
-          - jruby-9.3
+          - jruby
```

Comment:
Maybe -9.4 to prevent breakages when some early 9.5.0.x is out?

Reply:
This would also be in line with https://github.com/rubocop/rubocop/pull/11551/files#diff-b79dca49270186d1f5b31d431888b2811a16786f6d590f1a5220507d3de190cbR70

Reply:
> Maybe -9.4 to prevent breakages when some early 9.5.0.x is out?

Originally, there was no version designation, so I reverted to the original, but it certainly looks better to use 9.4.
I updated this PR.